app/furniture/marketplace/breadcrumbs.rb

```diff
@@ -98,3 +98,8 @@
   parent :marketplace_tax_rates, tax_rate.marketplace
   link "Edit Tax Rate '#{tax_rate.label}'", marketplace.location(:new, child: :tax_rate)
 end
+
+crumb :payment_settings do |marketplace|
+  parent :edit_marketplace, marketplace
+  link "Manage Payment Settings", marketplace.location(:index, child: :payment_settings)
```

Comment:
Can we use the value stored in `I18n.t("marketplace.payment_settings.index.link_to")` so that the text is consistent in the breadcrumb and the links to the breadcrumb?